CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.10.5.4.1
+- Fixes an issue in which InMobi event listeners were garbage collected.
```

Comment:
let's use past tense "Fixed"
and maybe "in which" -> "where"

Reply:
Sounds perfect. ✔️